omod/src/main/webapp/resources/scripts/angular-common.js

```diff
@@ -16,6 +16,14 @@ angular.module('uicommons.common', []).
 
         // to prevent the browser from displaying a password pop-up in case of an authentication error
         $httpProvider.defaults.headers.common['Disable-WWW-Authenticate'] = 'true';
+        
+	    // IE GET cache problem
+        if (!$httpProvider.defaults.headers.get) {
+            $httpProvider.defaults.headers.common = {};
```

Comment:
Do you really need to initialize this? Otherwise line18 would have already failed